torch/_lazy/computation.py

```diff
@@ -0,0 +1,15 @@
+import torch._C._lazy
+import torch._C._lazy_ts_backend
+
+def get_tensors_ts_device_data_node(tensors):
```

Comment:
I think anything that is ts_backend specific should be under the torch._lazy.ts_backend module.

Reply:
I thought about the same thing. The reason I'm putting the method here is we should generalize it for XLA soon. But I'm okay to move this to ts_backend.py before that . Just let me know.

Reply:
yea, I think ideally we want it at this level, and it is unclear if we can actually deliver that easily or if it will be a little different for xla.  but lets leave it here now and just add some comments explaining the proposal to add xla here, and that this is still 'unstable/prototype'